src/lib/compliance/tcpa.ts

```
@@ -0,0 +1,409 @@
+  
+  // Default for unknown
+  'DEFAULT': 'America/New_York'
+};
+
+// Area code to timezone mapping (for when state is unknown)
+const AREA_CODE_TIMEZONES: Record<string, string> = {
+  // Eastern
+  '201': 'America/New_York', '202': 'America/New_York', '203': 'America/New_York',
+  '212': 'America/New_York', '215': 'America/New_York', '216': 'America/New_York',
+  '240': 'America/New_York', '267': 'America/New_York', '301': 'America/New_York',
+  '302': 'America/New_York', '303': 'America/New_York', '304': 'America/New_York',
```

Biome (1.9.4)

[error] This property value named 303 is later overwritten by an object member with the same name.

Overwritten with this value.

If an object property with the same name is defined multiple times (except when combining a getter with a setter), only the last definition makes it into the object and previous definitions are ignored.
Unsafe fix: Remove this property value named 303


(lint/suspicious/noDuplicateObjectKeys)